data/products/sapcal/sle15/sp4/packages.yaml

```diff
@@ -1,2 +1,9 @@
 packages:
   _namespace_sapcal_libopenssl_old: Null
+  _namespace_sapcal_extra:
+    package:
+      - xmlstarlet
+      - libcap-progs
+      - openssl
+      - libicu60_2
```

Comment:
What does `libicu60_2` provide that is not part of `libicu-suse65_1` which is shipped from the ICU package and already part of the image?

Reply:
I don't know - I explicitly asked Antonia about that and she told me "Regarding package 'libicu60_2' I had to check with my colleague who needs all those libraries and he confirmed that it is important to have exactly package 'libicu60_2' cause it`s a requirement for his product."

Reply:
Saw the reply. Thanks.